<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="pipelineTreeDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Pipeline tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="pipelineTreeDockContents"/>
  </widget>
  <widget class="QDockWidget" name="pipelinePropertiesDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Pipeline properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="pipelinePropertiesDockContents"/>
  </widget>
  <widget class="QDockWidget" name="logViewerDock">
   <property name="windowTitle">
    <string>Log viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="logViewerDockContents"/>
  </widget>
   <widget class="QDockWidget" name="scriptingConsoleDock">
     <property name="windowTitle">
       <string>Scripting Console</string>
     </property>
     <attribute name="dockWidgetArea">
       <number>8</number>
     </attribute>
     <widget class="QWidget" name="scriptingConsoleDockContents"/>
   </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
